typings/named-asset-import.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module '*.svg' {
```

Comment:
copied from CRA

Reply:
Please rename the file to use camelcase as we do for other files that are not React components.

Reply:
Is there a standard naming convention for TS typings?

Reply:
The only standard i could find is that typescript declaration files are prefixed by lib.
```
TypeScript names these declaration files with the pattern lib.[something].d.ts. If you navigate into a file with that name, you can know that you're dealing with some built-in part of the platform, not user code.
```
Reference : https://microsoft.github.io/TypeScript-New-Handbook/chapters/type-declarations/

React seems to type their declaration files in lowercase with -
```
jsx-dev-runtime.d.ts
jsx-runtime.d.ts
```